File: src/codes/Users_Manual/link/visualizer_gpu.cpp
//~#include<autd3.hpp>
#include "autd3/link/visualizer.hpp"

//~int main() {
//~auto autd = autd3::ControllerBuilder()
//~                .add_device(autd3::AUTD3(autd3::Vector3::Zero()))
//~                .open(
autd3::link::Visualizer::builder().with_gpu(-1)
    //~                    );
    //~return 0; }

File: src/codes/Users_Manual/link/visualizer_gpu.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(
Visualizer.Builder().WithGpu(-1)
~);